test/distributed/org/apache/cassandra/distributed/action/GossipHelper.java

```diff
@@ -473,6 +473,7 @@ public static void withProperty(CassandraRelevantProperties prop, boolean value,
         withProperty(prop, Boolean.toString(value), r);
     }
 
+    // checkstyle: suppress below 'clearValueSystemPropertyUsage'
```

Comment:
The `checkstyle: suppress below ` uses `offCommentFormat` here [1], which means that all the checks below this comment are suppressed. This makes the checks wildly inaccurate, as a developer can still use `clearValue` and get no errors on build. 

Should we use `checkstyle: suppress nearby` for all such places?

[1] https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.html

Reply:
sure, good idea.